front/src/modules/ui/object/record-table/components/RecordTableHeader.tsx

```diff
@@ -101,7 +113,7 @@ const HIDDEN_TABLE_COLUMN_DROPDOWN_SCOPE_ID =
 const HIDDEN_TABLE_COLUMN_DROPDOWN_HOTKEY_SCOPE_ID =
   'hidden-table-columns-dropdown-hotkey-scope-id';
 
-export const RecordTableHeader = () => {
+export const RecordTableHeader = ({ tableBodyRef }: RecordTableHeaderProps) => {
```

Comment:
I don't think we should use tableBodyRef, I believe putting a ref on tableheader should be good enough to assess column sizes